packages/api-server/src/logFormatter/formatters.ts

```diff
@@ -125,7 +125,8 @@ export const formatMessage = (logData: any) => {
     pretty = chalk.white(msg)
   }
   if (level === 'warn') {
-    pretty = chalk.magenta(msg)
```

Comment:
Switched from magenta to orange here to align with the color we use for warnings in other places in Redwood